packages/event-handler/src/rest/BaseRouter.ts

```diff
@@ -76,6 +105,68 @@ abstract class BaseRouter {
     }
   }
 
+  /**
+   * Handles errors by finding a registered error handler or falling
+   * back to a default handler.
+   *
+   * @param error - The error to handle
+   * @returns A Response object with appropriate status code and error details
+   */
+  protected handleError(error: Error): Response {
+    const handler = this.errorHandlerRegistry.resolve(error);
+    if (handler != null) {
```

Comment:
Why not use strict checks here? I.e. `!==`

Reply:
I'm just being overly defensive here as this will catch  the case of `undefined` too. Will change it.

Reply:
Done